### DISCUSSION TITLE
How can I hide caption in images?

### DISCUSSION THREAD

**Discussion by javierfs** [sachinchoolur/lightGallery#1130] [Q&A] [answered]
Is there a way to hide the captions and the background rectangle (i.e., appendSubHtmlTo, lg-sub-html)?

**Comment by sachinchoolur** [comment, marked as answer]
hey @javierfs,

If you don't provide `lg-sub-html`, captions won't appear.
Are you looking for hiding caption in any particular scenario?

**Reply by javierfs** [reply]
it seems the caption overlaps always the image and I want to show, for instance, the X-Axis of my graphs and its ticks. I think I managed hiding it by removing the lg-sub-html. It would be nice if there is an option to show the caption just below the image without overlapping it :D

**Reply by sachinchoolur** [reply]
If you set `allowMediaOverlap:false`, captions won't overlap the image as you can see on the [homepage](https://www.lightgalleryjs.com/#lg=nature&slide=2)

If you always want to show the captions, just bewlow the image, you can use the [relativeCaption](https://github.com/sachinchoolur/lightGallery/issues/1046) plugin.
I haven't created documentation and demos for relativeCaption plugin. Hope you'll be able to use it using the info shared in the above link 

let me know if you have any questions
